src/__xsg/info.lua

```diff
@@ -80,11 +80,26 @@ if _info.snowgemd == "running" then
             end
         end,
         function () -- synchronization check
-            local _exitcode = _exec_xsg_cli("getblocktemplate")
-            _info.synced = _exitcode == 0
-            if not _info.synced then
+            local _exitcode, _stdout, stderr = _exec_xsg_cli("getblockchainsyncstatus")
+            local _success, _output = _get_xsg_cli_result(_exitcode, _stdout, _stderr)
+
+            if _success then
+                if _output.IsBlockchainSync = true
+                    _info.level = "ok"
```

Comment:
`_info.level` has to be set to `ok` only once and that is before any of checks happen. (It `ok` by default.) So not here 🙂

Reply:
Please remove others `_info.level = "ok"` after this one as well.